rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -103,12 +103,19 @@ def test_exec(config):
             cmd = 'radosgw-admin zone placement modify --rgw-zone=%s ' \
                   '--placement-id=default-placement --compression=%s' % (zone,compression_type)
             out = utils.exec_shell_cmd(cmd)
+            ceph_version = utils.exec_shell_cmd("ceph version")
+            ceph_ver_op = ceph_version.split()
             try:
                 data = json.loads(out)
-                if data['placement_pools'][0]['val']['storage_classes']['STANDARD']['compression_type'] == compression_type:
-                    log.info('Compression enabled successfully')
-                else:
-                    raise ValueError('failed to enable compression')
+                for i in ceph_ver_op:
+                    if i == 'luminous':
+                        if data['placement_pools'][0]['val']['compression'] == compression_type:
+                            log.info('Compression enabled successfully')
+                    else:
+                        if i == 'nautilus' or i == 'octopus':
```

Comment:
instead of performing  `or`, it could be written as `i in ['nautilus', 'octopus']`

Reply:
done. please look and let me know.